scripts/checkForUpdates.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+VERSION_FILE="https://raw.githubusercontent.com/fuseio/fuse-network/master/Version"
+
+function updateContainers {
+  "$DIR/quickstart.sh"
+}
+
+function getContainerImageNames {
+  docker ps -a | awk '{ print $2 }'
+}
+
+function grabAndParseVersionFile {
+  wget -O "$DIR/versionFile" $VERSION_FILE
+  export $(grep -v '^#' "$DIR/versionFile" | xargs)
+  
+  # Print versions
+  echo "Oracle version = $DOCKER_IMAGE_ORACLE_VERSION"
+  echo "Parity version = $DOCKER_IMAGE_FUSE_PARITY_VERSION"
+  echo "Fuse app version = $DOCKER_IMAGE_FUSE_APP_VERSION"
+  echo "Netstats version = $DOCKER_IMAGE_NET_STATS_VERSION"
+}
+
+function versionComp {
+  local nodeName=$1
+  local expectedVersion=$2
+  if [[ $nodeName == *"$expectedVersion"* ]]; then
+    return 0
+  else
+    return 1
+  fi
+  
+  return 0
+}
+
+function checkContainers {
+  grabAndParseVersionFile
+  conatiners=$(getContainerImageNames)
+  update=0
+  
+  for IMAGE_NAME_WITH_TAG in ${conatiners[@]}; do
```

Comment:
What I don't like about this approach is that it does not take into account that somebody might run any containers except the ones that are run by `quickstart.sh`. But if this is a known restiction - then we should be fine.

Reply:
I think it should be a restriction since the quickstart sets the links between containers having a frankenstein config is probably not the way to go